actions/universal-remove-banners.js

```diff
@@ -1,18 +1,24 @@
 const cheerio = require('cheerio');
```

Comment:
This shouldn't have any changes except for the `description-quicklinks` grandchild. Can we make sure these changes don't affect anything else?

Reply:
Should be fine